### PR TITLE
Added command to clear verified numbers

### PR DESCRIPTION
**Pull request by kkrampa**
@czue @gcapalbo 
After yesterday PR (https://github.com/dimagi/commcare-hq/pull/5917) task that I wrote still raises exception. Could we clear clear verified numbers in all logistics domains?

### CODE REVIEW

File: custom/logistics/management/commands/clear_verified_numbers.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from corehq.apps.sms.mixin import VerifiedNumber
+from custom.ewsghana.models import EWSGhanaConfig
+from custom.ilsgateway.models import ILSGatewayConfig
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        logistics_domains = ILSGatewayConfig.get_all_enabled_domains() + EWSGhanaConfig.get_all_enabled_domains()
+        for domain in logistics_domains:
```
Comment (gcapalbo) [inline custom/logistics/management/commands/clear_verified_numbers.py]: This makes me nervous - can't we just make this command take a list of domains as args to do this for?

**Comment by gcapalbo** [comment]
@kkrampa This is fine to do to start with a clean slate but I think this is going to keep happening until the comment I made here is addressed: https://github.com/dimagi/commcare-hq/pull/5809#discussion_r26577420

**Comment by kkrampa** [comment]
@gcapalbo This task doesn't work simultaneously for multiple users so that shouldn't be problem.

**Comment by kkrampa** [comment]
Updated

File: custom/logistics/management/commands/clear_verified_numbers.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.core.management import BaseCommand
+from corehq.apps.sms.mixin import VerifiedNumber
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        logistics_domains = ['ewsghana-test-1', 'ilsgateway-test-1', 'ilsgateway-test-2']
+        for domain in logistics_domains:
```
Comment (gcapalbo) [inline custom/logistics/management/commands/clear_verified_numbers.py]: @kkrampa I was thinking more along the lines of:

    for domain in args:
        ...

that way we can use it again for any domains in the future too (and we can also move it to corehq/apps/sms/management/commands). I just wanted to make sure we had a way to explicitly list the domains so we don't have any mix-up.

**Comment by gcapalbo** [comment]
@kkrampa Ok, thanks for confirming.  Does this task happen to run simultaneously for multiple domains?  In that case it still might cause an issue if there's an overlap in phone numbers between the domains.

**Comment by kkrampa** [comment]
@gcapalbo Task is run separately for particular domains.

**Comment by gcapalbo** [comment]
@kkrampa Sorry, I didn't get notified when this was last updated so I thought we were still waiting. Do you still need this? It looks fine now so if you reopen I'll merge.

**Comment by kkrampa** [comment]
Hi, I don't need this anymore, so we can leave this closed.

**Pull request closed without merging**